packages/interactivity-router/src/head.ts

```diff
@@ -29,6 +34,14 @@ export const updateHead = async ( newHead: HTMLHeadElement[] ) => {
 		}
 	}
 
+	await Promise.all(
```

Comment:
Another option we could try is to move this code to its own function. This way, `updateHead()` would become synchronous again, and we wouldn't have to use `await` inside `renderRegions()`.